Dockerfile

```diff
@@ -3,19 +3,20 @@ MAINTAINER m3adow
 
 # Long command to keep the FS deltas small
 RUN apk --update add python \
-		python-dev \
-		py-setuptools \
-		py-pip \
-		openssl-dev \
-		openssl \
-		musl-dev \
-		gcc \
-		libffi-dev \
-		darkhttpd \
-	&& wget -qO- https://codeload.github.com/zenhack/simp_le/tar.gz/master | tar xz \
-	&& pip install -e /simp_le-master/ \
-	&& mkdir /certs \
-	&& apk --purge del musl-dev openssl-dev libffi-dev gcc python-dev py-pip
+        python-dev \
+        py-setuptools \
+        py-pip \
+        openssl-dev \
+        openssl \
+        musl-dev \
+        gcc \
+        libffi-dev \
+        git \
+        darkhttpd \
+    && git clone https://github.com/zenhack/simp_le.git /simp_le-master \
```

Comment:
Can we clone a specific commit/tag here?

Reply:
Do you want to grab the latest 0.15.0 release tag here?

Reply:
That works for me.

Reply:
The following is probably easiest to do.

```
git clone --single-branch --branch 0.15.0 https://github.com/zenhack/simp_le.git /simp_le-master
```